### ISSUE TITLE
[Alchemy] Reverting a potion to its previous state rewards experience (glowstone redstone loop)

### ISSUE THREAD

**Issue by bm01** [mcMMO-Dev/mcMMO#2652]
The current code allows the prevention of experience gain from the "Potion II" <-> "Potion (extended)" conversion (glowstone <-> redstone).

Some potions however don't possess an extended variant, brewing them with a redstone turn them into their base variant instead (potion II -> potion).

When that happens, mcMMO sees that the output is at a different brewing stage than the input and rewards experience. It is also possible to upgrade it again with a glowstone and gain experience from it a second time.

In short, glowstone <-> redstone loop alternatively rewards tier 4 and tier 3 experience for some (splash) potions. Both are very common ingredients and require minimal effort.

Downgrading shouldn't reward experience. Detecting a downgraded potion and preventing its upgrade from rewarding experience isn't realistic though (or is it?).

**Comment by CeIebrimbor** [comment]
Not a bug. Players are using items to make a potions. This is how you get Alchemy exp. Each item used is being eaten by their loop. Just because they found an efficient way to use the system does not mean its a bug. They aren't even getting potion out of this. They are wasting items for exp.

**Comment by bm01** [comment]
They are getting the base (non upgraded / extended) variant of the potion.

That loop doesn't give experience if done on most of the potions. There's no reason why downgrading a potion should reward experience if converting an extended one into its upgraded variant (or the opposite) does.

**Comment by t00thpick1** [comment]
With 1.9 potions having a slight overhaul this is no longer relevant.
